templates/index.html

```diff
@@ -1,144 +1,48 @@
-          <button class="ctrlc" onclick="copy('https://random-word-api.herokuapp.com/word?number=10')">
-            Ctrl+C
-          </button>
-        </p>
-      </div>
-
-      <div>
-        <p class="code">?length</p>
-        Set length of requested words. This will only return words
-        that contain x amount of letters. <br />
-        Example:
-
-        <p class="url"><a
-            href="https://random-word-api.herokuapp.com/word?length=5">https://random-word-api.herokuapp.com/word?length=5</a>
-          <button class="ctrlc" onclick="copy('https://random-word-api.herokuapp.com/word?length=5')">
-            Ctrl+C
-          </button>
-        </p>
-      </div>
-
-      <div>
-        <p class="code">?lang</p>
-        Set language of requested words. Go to Github to get more info on how to
-        add your own language. <br />
-        Currently supported languages: 
-        <span class="code" id="langs">
-          <a href="https://random-word-api.herokuapp.com/languages">click</a>
-        </span> <br />
-        Example:
-
-        <p class="url"><a
-            href="https://random-word-api.herokuapp.com/word?lang=es">https://random-word-api.herokuapp.com/word?lang=es</a>
-          <button class="ctrlc" onclick="copy('https://random-word-api.herokuapp.com/word?lang=es')">
-            Ctrl+C
-          </button>
-        </p>
-      </div>
-
-    </div>
-
-    <div id="license">
-      <div>
-        Collection of random words I spent way too long scraping from websites.
-        I didn't find any random word API that was just a request. You can get
-        the
-        <a href="https://github.com/RazorSh4rk/random-word-api">source</a> and
-        host it yourself if you want. It is now running on a hobby dyno, so it
-        should be up quite reliably.
-
-        <br />
-        <a href="https://choosealicense.com/licenses/wtfpl/#">Runs under the DWTFYW license.</a>
-      </div>
-    </div>
-
-    <div id="donate">Not yet but thanks for checking.</div>
-  </div>
-  
-  <script src="/js"></script>
-
-</body>
-
+<html lang="en-US">
+    <head>
+        <title>Random Word API</title>
+        <meta charset="utf-8">
+        <meta name="phone" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+        <link rel="stylesheet" href="index.css" type="text/css">
+        <noscript><meta http-equiv="refresh" content="0; url=error.html"/></noscript>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="index.js"></script>
+    </head>
+    <body>
+        <div class="nav">
+            <h2>Random Word API</h2>
+            <h3 onclick="endpoints();">Endpoints</h3>

```

Comment:
If you could have some pointer that im looking at >this< page right now that would make this perfect